src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -193,17 +198,20 @@ def parse_netcore_version(file_path):
 
 
 def parse_node_version(file_path):
+    # from node experts the node value in package.json can be found here   "engines": { "node":  ">=10.6.0"}
     import json
     import re
+    version_detected = []
     with open(file_path) as data_file:
-        data = []
-        for d in find_key_in_json(json.load(data_file), 'node'):
-            non_decimal = re.compile(r'[^\d.]+')
-            # remove the string ~ or  > that sometimes exists in version value
-            c = non_decimal.sub('', d)
-            # reduce the version to '6.0' from '6.0.0'
-            data.append(c[:3])
-        version_detected = sorted(data, key=float, reverse=True)
+        data = json.load(data_file)
+        for key, value in data.items():
+            if key == 'engines' and 'node' in value:
+                value_detected = value['node']
+                non_decimal = re.compile(r'[^\d.]+')
+                # remove the string ~ or  > that sometimes exists in version value
+                c = non_decimal.sub('', value_detected)
+                # reduce the version to '6.0' from '6.0.0'
```

Comment:
Either this comment or the code below is not right. `6.0` is only 3 characters long

Reply:
Here is how i understand Python's slice notation 
c[:4] => this reads as :stop =>items from the beginning through stop-1 , so this is until character 3 so will change 6.0.0 to 6.0

Reply:
checked below out. 
```bash
Python 2.7.5 (default, Jul 13 2018, 13:06:57)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-28)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> '6.0.0'[:4]
'6.0.'
>>> '6.0.0'[:3]
'6.0'
```

Reply:
weird that's not what i am seeing i tried printing out the values on console

![image](https://user-images.githubusercontent.com/13839372/53924069-672aa780-402f-11e9-9a23-f90a7dfbe74b.png)

Reply:
Also saw this article explaining it

https://stackoverflow.com/questions/509211/understanding-slice-notation

i am concerned though, that the behavior is going to be different with different Python versions.

Reply:
Figured out the issue here with 10.0.0 vs 6.0.0 ,so changed this logic